aws/resource_aws_pinpoint_app.go

```diff
@@ -193,6 +207,12 @@ func resourceAwsPinpointAppUpdate(d *schema.ResourceData, meta interface{}) erro
 		return err
 	}
 
+	if err := setTagsPinPointApp(conn, d); err != nil {
+		return fmt.Errorf(
+			"Error Updating PinPoint App tags: \"%s\"\n%s",
```

Comment:
We prefer to not introduce extra newlines into our error messaging:

```go
		return fmt.Errorf("error updating PinPoint Application (%s) tags: %s", d.Id(), err)
```

Reply:
Make sense, Implemented 👍